### ISSUE TITLE
Colors!

### ISSUE THREAD

**Issue by mrocklin** [dask/dask-sphinx-theme#76]
Thanks all for the work on the sphinx theme.  I'm curious about how we're choosing colors here.  For example in our configuration reference I'm seeing the following:

![image](https://user-images.githubusercontent.com/306380/172953923-ef2d5a84-5ae4-47b8-829e-fa45fc015441.png)

Are these intentional (from a pallete somewhere?) or are we just deferring to the underlying theme?  If the latter, should I just make modifications as I see fit, or do we want to consult something grander (if anyone else wants to take this away from me I'd be happy for it)

**Comment by jrbourbeau** [comment]
I think colors are intentional and coming from https://www.dask.org/brand-guide (though maybe we haven't gotten to updating colors in all parts of the theme yet)

cc @scharlottej13 as she'll know more about this

**Comment by mrocklin** [comment]
I don't know.  That background blue looks awfully like standard Sphinx.  My experience doing Sphinx style guide stuff is that there are a lot of little settings you don't catch the first time around.

**Comment by mrocklin** [comment]
(or at least, I failed at finding everything the first few times I did this)

**Comment by scharlottej13** [comment]
> I think colors are intentional and coming from https://www.dask.org/brand-guide (though maybe we haven't gotten to updating colors in all parts of the theme yet)

Yes, exactly! I met w/ Jeromy and he also sent me these additional style guides:
[Dask-Doc Portal Spec Sheet.pdf](https://github.com/dask/dask-sphinx-theme/files/8874326/Dask-Doc.Portal.Spec.Sheet.3.pdf)
[Dask-Doc Portal mockup.pdf](https://github.com/dask/dask-sphinx-theme/files/8874327/Dask-Doc.Portal.mockup.1.pdf)

> I don't know. That background blue looks awfully like standard Sphinx. My experience doing Sphinx style guide stuff is that there are a lot of little settings you don't catch the first time around.

Yup, spot on :). I went through and tried to find how to change most things, but I'm sure there were a bunch of little things I missed. I would be happy for you or others to jump in and make modifications. I outlined a couple outstanding items in #67 too and then there's a heated discussion on footer background colors in #75.

**Comment by scharlottej13** [comment]
some options inspired by the spec sheet and discussion w/ @mrdanjlund77

**option 1: black/light grey/slightly darker light grey**

![Screen Shot 2023-04-06 at 3 52 56 PM](https://user-images.githubusercontent.com/8620816/230508810-c74b2816-821c-4fd8-a4aa-b1782e849078.png)

---

**option 2: dark blue/slightly darker light grey**

![Screen Shot 2023-04-06 at 4 02 45 PM](https://user-images.githubusercontent.com/8620816/230513301-08619ba0-1c47-46ae-96d1-66d67cbb2d14.png)

---

**option 3: dark blue text, light blue background + light grey background**

![Screen Shot 2023-04-06 at 4 37 54 PM](https://user-images.githubusercontent.com/8620816/230513272-32eff5e5-c203-4b3b-b2ad-2103ee7d8878.png)

**Comment by jrbourbeau** [comment]
Nice! 

My personal ranking would be (in order of decreasing preference)

- Option 2
- Option 1
- Option 3

@jacobtomlinson I'd be curious if you have any thoughts here

**Comment by jacobtomlinson** [comment]
Yup I agree with that ranking.
